packages/create-pantheon-decoupled-kit/src/types.ts

```diff
@@ -46,6 +46,10 @@ export interface DecoupledKitGenerator<Prompts extends Answers> {
 	 * Any message(s) to be rendered after actions are successfully completed.
 	 */
 	nextSteps?: string[];
+	/**
+	 * Set to true if the generator is to use the experimental app directory from  Next JS.
+	 */
+	appDir?: boolean;
```

Comment:
This doesn't need to be added to the types. Use the `data` field in the generator to pass arbitrary data to the templates.

Reply:
Ah good catch, I had confused this with the `addOn` flag originally and meant to remove this.